plugins/storm.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: storm
+spec:
+  version: v0.0.1
+  homepage: https://github.com/guilhem/kubectl-storm
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/guilhem/kubectl-storm/releases/download/v0.0.1/kubectl-storm_v0.0.1_darwin_amd64.tar.gz
+    sha256: 238773f55ab2d34eb228d0bdd214f62d793366f2d1c1a5b2e10ff00aa0cbe270
+    bin: kubectl-storm
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: arm64
+    uri: https://github.com/guilhem/kubectl-storm/releases/download/v0.0.1/kubectl-storm_v0.0.1_darwin_arm64.tar.gz
+    sha256: 49b35aede7483c2e4053c55ea5c3bbccbd870bb8d8a50f74f99467a072cb027b
+    bin: kubectl-storm
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/guilhem/kubectl-storm/releases/download/v0.0.1/kubectl-storm_v0.0.1_linux_amd64.tar.gz
+    sha256: 3801eefe077dc88714dad1141b77924cda5546e318d6c1915957f27a437c5a9e
+    bin: kubectl-storm
+  shortDescription: Detect update storm
+  description: |
+    Usage:
+      kubectl storm
+      This plugin detect changes on resources.
+      Read more documentation at: https://github.com/guilhem/kubectl-storm
```

Comment:
We print all this after the plugin is installed. So please delete this and explain what the plugin does instead.

Reply:
Ok :)
I just follow example from https://github.com/rajatjindal/krew-release-bot?tab=readme-ov-file#basic-setup
https://github.com/rajatjindal/kubectl-evict-pod/blob/main/.krew.yaml